python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +861,70 @@ def run(self, inputDefects, camera):
         return pipeBase.Struct(
             mergedDefects=merged,
         )
+
+# Subclass the MergeDefects task to reduce the input dimensions
+# from ("instrument", "detector", "exposure") to
+# ("instrument", "detector").
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
+                                      dimensions=("instrument", "detector")):
+    inputDefects = cT.Input(
```

Comment:
This is a minor point, and may be irrelevant.  But now that we have different dimensions, I wonder if the name here should be `darkDefects` and the secondary should be `flatDefects`?

Reply:
I like long and descriptive names, like the ones that are already there: `cpPartialDefectsFromDarkCombined` and `cpPartialDefectsFromFlatCombinedWithFilter`. But I would be happy to change them to what you suggest if you believe that's better. Please let me know.

Reply:
I mean the name on the left side of the equation, `inputDefects` -> `inputDarkDefects` which I think is more descriptive, and below `secondaryInputDefects` -> `inputFlatDefects`.

Reply:
Ah, I see! Yeah, I can change that :)